examples/geometry_cube_wx.py

```diff
@@ -0,0 +1,48 @@
+"""
+Example showing multiple rotating cubes. This also tests the depth buffer.
+"""
+
+import imageio
+import pygfx as gfx
+
+from PyQt5 import QtWidgets
```

Comment:
Filename says this is a wx example but it uses qt :)

Reply:
I think I made a mistake when copying the example code :)  I just deleted this file for now.